src/KingSystem/Resource/resResourceAwareness.h

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#include <agl/Utils/aglParameter.h>
+#include <agl/Utils/aglParameterObj.h>
+#include "KingSystem/Resource/resResource.h"
+#include "KingSystem/Utils/ParamIO.h"
+
+namespace ksys::res {
+
+class Awareness : public ParamIO, public Resource {
+    SEAD_RTTI_OVERRIDE(Awareness, Resource)
+public:
+    Awareness();
+    ~Awareness() override = default;
+
+    bool needsParse() const override { return true; }
+    bool ParamIO_m0() override { return true; }
+
+private:
+    void doCreate_(u8*, u32, sead::Heap*) override {}
+    bool parse_(u8* data, size_t size, sead::Heap* heap) override;
+
+    agl::utl::ParameterObj mBasisObj;
+    agl::utl::ParameterObj mSightObj;
+    agl::utl::ParameterObj mHearingObj;
+    agl::utl::ParameterObj mSenseObj;
+    agl::utl::ParameterObj mWorryObj;
+
+    agl::utl::Parameter<s32> contactpoint_num;
```

Comment:
Could you add `public:` before this line to make all the parameters public? Otherwise there's no way to actually access the Awareness parameters

(or add getters like res::GameData, I guess, but that's more work)